Tests/UIKitTests/UICollectionViewExtensionsTests.swift

```diff
@@ -75,6 +92,41 @@ final class UICollectionViewExtensionsTests: XCTestCase {
     }
     #endif
 
+    func testSafeScrollToIndexPath() {
+        let validIndexPathTop = IndexPath(row: 0, section: 0)
+
+        flowLayoutCollectionView.contentOffset = CGPoint(x: 0, y: 30)
+        XCTAssertNotEqual(flowLayoutCollectionView.contentOffset, .zero)
+
+        flowLayoutCollectionView.safeScrollToItem(at: validIndexPathTop, at: .top, animated: false)
+        XCTAssertEqual(flowLayoutCollectionView.contentOffset, .zero)
+
+        let validIndexPathBottom = IndexPath(row: 4, section: 0)
+
+        let bottomOffset = CGPoint(x: 0, y: flowLayoutCollectionView.collectionViewLayout.collectionViewContentSize.height - flowLayoutCollectionView.bounds.size.height)
+
+        flowLayoutCollectionView.contentOffset = .init(x: 0, y: 30)
+        XCTAssertNotEqual(flowLayoutCollectionView.contentOffset, bottomOffset)
+
+        flowLayoutCollectionView.safeScrollToItem(at: validIndexPathBottom, at: .bottom, animated: false)
+        #if os(tvOS)
+        XCTAssertEqual(bottomOffset.y, flowLayoutCollectionView.contentOffset.y, accuracy: 2.0)
```

Comment:
Why do you need any accuracy at all? You can't get the exact value?

Reply:
I didn't actually test them without the accuracy parameter. But, since the table view methods were using it and these are mostly adapted copies from from there, the calls were kept.
I'll run the tests with the exact values on iOS and tvOS and update this code accordingly.